run_dense_net.py

```diff
@@ -101,6 +101,15 @@ def get_train_params_by_name(name):
     parser.add_argument(
         '--not-renew-logs', dest='renew_logs', action='store_false',
         help='Do not erase previous logs for model if exists.')
+    
+    parser.add_argument(
+        '--num_inter_threads', '-inter', type=int, default=1, metavar='',
```

Comment:
by tensorflow config default these values should be 0. Maybe it's better to leave default values same that tensorflow provides?

Reply:
inter and intraop threads are used by CPU's only and atleast for densenet for CPU models, these provide better results. This might change in future as the model evolves. (For google, they just provided a mechanism to take these as arguments and hence the default as 0. I would keep it. Hope it helps.

Reply:
Thank you for your clarification and nice PR!